src/main/java/ch/njol/unofficialmonumentamod/UnofficialMonumentaModClient.java

```diff
@@ -124,9 +156,10 @@ public void onInitializeClient() {
 
 	public static void onDisconnect() {
 		abilityHandler.onDisconnect();
-		Notifier.onDisconnect();
 		LocationNotifier.onDisconnect();
 		spoofer.onDisconnect();
+		//TODO if false, re-schedule unload.
```

Comment:
Does this TODO still need to be addressed?

Reply:
indeed.